src/main/webapp/gameHelper.js

```diff
@@ -179,3 +185,31 @@ function changeThreshold() {
   fetch('/promotion-threshold', {method: 'POST', body: params});
   window.location='promotionquiz.html';
 }
+
+/*
+ * This function splits sentences into words the current dialogue displayed and searches for PLAYER_NAME_PARAMETER
```

Comment:
This comment doesn't make sense (missing words? extra words?)

"splits the sentences into words the current dialogue displayed" <-- I don't know what this means